asab/log.py

```diff
@@ -153,7 +153,7 @@ async def schedule(app, interval):
 					elif format == '5micro':
 						self.SyslogHandler.setFormatter(SyslogRFC5424microFormatter(sd_id=Config["logging"]["sd_id"]))
 					elif format == 'json':
-						self.SyslogHandler.setFormatter(JSONFormatter(sd_id=Config["logging"]["sd_id"]))
+						self.SyslogHandler.setFormatter(JSONFormatter())
```

Comment:
json format is configured in [logging:syslog] config section. But it is not a syslog, isn't it? What is it?

Reply:
that's ok